ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/validation/CommitterModelConformanceValidator.scala

```diff
@@ -108,19 +112,21 @@ private[transaction] class ModelConformanceValidator(engine: Engine, metrics: Me
       } yield ()
       stepResult.fold(identity, _ => StepContinue(transactionEntry))
     } catch {
-      case missingInputErr @ Err.MissingInputState(key) =>
+      case missingInputErr @ Err.MissingInputState(
+            key
+          ) => // Missing package or contract not specified as input
         logger.error(
-          "Model conformance validation failed due to a missing input state (most likely due to invalid state on the participant).",
+          "Model conformance validation using on-ledger data failed due to a missing input state (most likely due to an invalid participant state).",
           missingInputErr,
         )
         rejections.reject(
           transactionEntry,
           Rejection.MissingInputState(key),
           commitContext.recordTime,
         )
-      case err: Err =>
+      case err: Err => // Archive decoding error or other bug
         logger.error(
```

Comment:
```suggestion
        logger.warn(
```

Reply:
This is actually an internal error, i.e. the participant really did something wrong. Why are you suggesting to log it at `warn`?